packages/theme/src/cli/utilities/theme-selector/filter.ts

```diff
@@ -27,7 +27,15 @@ function filterByTheme(store: string, themes: Theme[], filter: Filter) {
     const error = `The ${store} store doesn't have a theme with the "${identifier}" ID or name`
 
```

Comment:
One thing I noticed during tophat was that if we input a pattern with no matches, the err message is as follows - I think we can use this opportunity to update the message since we are now accepting patterns
`The quickstart-b88db4f0.myshopify.com store doesn't have a theme with the "f*" ID or name`       

Something along these lines is what I'm thinking, though we could add more specificity if you'd like

```suggestion
    const error = `No themes on the store ${store} match the identifier "${identifier}"`
```